app/views/users/show.html.erb

```diff
@@ -1,3 +1,30 @@
-<h1>
-  This is the list of a user with id
-</h1>
\ No newline at end of file
+<%= render "shared_view/user", user: @user %>
+
+<div class="bio">
+  <h2>Bio</h2>
+  <%= @user.bio %>
+</div>
+<div class="posts">
+  <ul>
+    <% @user.user_recent_posts.each_with_index do |post, index| %>
+      <li>
+       <h3>Post: #<%=  index + 1 %></h3>
+        <div>
+          <%= truncate(post.text, length: 150, omission: '...') %>
+          <span>
+            <%= link_to "See more", user_post_path(@user, post) %>
+          </span>
+        </div>
+        <div class="user_comments">
+          <h4>Comments: x,</h4>
+          <h4>Likes: y</h4>
```

Comment:
- [ ] Please consider using the actual counters for these values! 🤓 🔧 

![image](https://github.com/touredri/blog/assets/77069011/64fbd7e5-a39c-4dce-9f52-046dff0f68cd)

Reply:
@anagudelogu  Ok